valhalla/jawn/src/managers/apiKeys/KeyManager.ts

```diff
@@ -215,7 +226,7 @@ export class KeyManager extends BaseManager {
   async getProviderKeys(): Promise<Result<any[], string>> {
     try {
       const result = await dbExecute(
-        `SELECT *
+        `SELECT *, COALESCE(byok_enabled, false) as byok_enabled
```

Comment:
can i be annoying and suggest if you do this, you also just enumarte all the other columns